frontend/src/components/housing/enquiry/EnquiryForm.vue

```diff
@@ -55,18 +56,23 @@ const initialFormValues: Ref<any | undefined> = ref(undefined);
 const showCancelMessage: Ref<boolean> = ref(false);
 
 // Form validation schema
-const stringRequiredSchema = string().required().max(255);
-
 const intakeSchema = object({
   enquiryType: string().oneOf(ENQUIRY_TYPE_LIST).label('Submission type'),
   submittedAt: date().required().label('Submission date'),
   relatedActivityId: string().nullable().min(0).max(255).label('Related submission'),
-  contactFirstName: stringRequiredSchema.label('First name'),
-  contactLastName: stringRequiredSchema.label('Last name'),
-  contactApplicantRelationship: string().required().oneOf(PROJECT_RELATIONSHIP_LIST).label('Relationship to project'),
-  contactPreference: string().required().oneOf(CONTACT_PREFERENCE_LIST).label('Contact Preference'),
-  contactPhoneNumber: stringRequiredSchema.label('Phone number'),
-  contactEmail: string().matches(new RegExp(Regex.EMAIL), 'Email must be valid').required().label('Email'),
+  contacts: array().of(
+    object({
+      contactApplicantRelationship: string()
+        .required()
+        .oneOf(PROJECT_RELATIONSHIP_LIST)
+        .label('Relationship to project'),
+      contactPreference: string().oneOf(CONTACT_PREFERENCE_LIST).label('Preferred contact method'),
+      email: emailValidator('Contact email must be valid').required().label('Contact email'),
+      firstName: string().required().max(255).label('Contact first name'),
+      lastName: string().required().max(255).label('Contact last name'),
+      phoneNumber: string().required().label('Contact phone number')
+    })
+  ),
```

Comment:
Should use the common contact validator rules in the validators folder like in `SubmissionIntakeForm`.